CMakeLists.txt

```
@@ -228,18 +230,19 @@ list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 if(PKG_CONFIG_FOUND)
     pkg_check_modules(ZLIB zlib)
     list(APPEND LXW_PRIVATE_INCLUDE_DIRS ${ZLIB_INCLUDE_DIRS})
-else(NOT ZLIB_FOUND)
+else()
```

So the original intention might have been:
```suggestion
endif()
if(NOT ZLIB_FOUND)
```
(and similar below).